src/platform/lumin-runtime/elements/builders/progress-bar-builder.js

```diff
@@ -53,7 +53,7 @@ export class ProgressBarBuilder extends UiNodeBuilder {
     setProgressColor (element, oldProperties, newProperties) {
         const { beginColor, endColor } = newProperties.progressColor;
         if ( beginColor !== undefined && endColor !== undefined ) {
-            element.setProgressColor( beginColor, endColor );
+            element.setProgressColor( ColorProperty.parse(beginColor), ColorProperty.parse(endColor) );
```

Comment:
what happens if the begin/end Colors are not valid?

Reply:
All defined properties are automatically validated before their values are updated. See ElementBuilder.update(...):  https://github.com/magic-script/magic-script-components-lumin/blob/d5e4f21911313d6e333aae3372500334c5f11278/src/platform/lumin-runtime/elements/builders/element-builder.js#L9